### PR TITLE
Removed flexible fleet wait time from travel time reporter

### PR DESCRIPTION
**Pull request by JoeJimFlood**
## Proposed changes

This removes the initial wait time for flexible fleets from the travel time reporter.

## Impact

More destinations will be available within a trip of a specified time by microtransit or NEV.

## Types of changes

What types of changes does your code introduce to ABM?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## How has this been tested?

Please describe the tests that you ran to verify your changes.

No tests were run

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

**Review by aber-sandag — APPROVED** [review]

File: src/main/python/TravelTimeReporter.py [hunk 1]
```diff
@@ -392,7 +392,7 @@ def get_ff_time(self, nev = False):
                 np.maximum(
                     self.constants[flavor + "DiversionConstant"] + direct_time,
                     self.constants[flavor + "DiversionFactor"] * direct_time
-                ) + self.constants[flavor + "WaitTime"],
+                )
```
Comment (aber-sandag) [inline src/main/python/TravelTimeReporter.py:395]: Still need to include comma here for np.maximum arguments

**Review by aber-sandag — CHANGES_REQUESTED** [review]

File: src/main/python/TravelTimeReporter.py (hunk 1 above)
Comment (aber-sandag) [reply src/main/python/TravelTimeReporter.py:395]: np.where arguments, not np.maximum

Comment (JoeJimFlood) [reply src/main/python/TravelTimeReporter.py:395]: Thanks! I have fixed it.

Comment (aber-sandag) [reply src/main/python/TravelTimeReporter.py:395]: Comma needs to be before self.settings["infinity"]

**Review by aber-sandag — CHANGES_REQUESTED** [review]

**Comment by JoeJimFlood** [comment]
Fixed (this is why I'm a modeler and not a newspaper editor)

**Review by aber-sandag — APPROVED** [review]

**Pull request merged**